packages/code-du-travail-frontend/pages/_app.js

```diff
@@ -26,7 +25,7 @@ if (typeof window !== "undefined") {
       );
     })
     .catch((err) => {
-      notifySentry(418, err.message || "Failed to load web component");
+      Sentry.captureException(err.message || "Failed to load web component");
```

Comment:
Ici soit on passe 
`Sentry.captureException(err)`
Soit 
`Sentry.captureException({message: err.message || "Failed to load web component", status: 418})`

Reply:
Je sais pas pourquoi 418...